### ISSUE TITLE
[Accurate and fast PPU issue] Suzuka 8 Hours (Japan) (En) - attract mode doesn't start [bsnes v113.1]

### ISSUE THREAD

**Issue by Max833** [byuu/bsnes#254]
It's not always the case, but sometimes the attract mode doesn't work:

https://streamable.com/94stl

**Edit:** I don't know why, but even when I start bsnes with accurate settings and dsp1b.rom, bsnes says it's HLE.

Same issue with "Suzuka 8 Hours (USA)".

It works fine with SNES9x and Mesen-S

**Comment by byuu** [comment]
With or without HLE?

**Comment by byuu** [comment]
Let's have someone confirm whether this happens or not on real hardware.

We also need someone who can set up the game in LLE mode to test and see if it's just an HLE issue or not.

**Comment by byuu** [comment]
It only happens with HLE. Tried LLE 20 times with entropy set to none+low+high and it always starts.

I reverted a gamepak change that should allow game ROMs to load DSP LLE again.

I'm not really interested in investing a lot of time fixing whatever's wrong with the HLE implementation that works right in Snes9X. HLE is a hack. It's probably slight CPU timing differences between the two emulators. Mesen-S doesn't have an HLE mode as far as I know.
